openpgp/packet/private_key.go

```diff
@@ -366,6 +374,24 @@ func (pk *PrivateKey) parseElGamalPrivateKey(data []byte) (err error) {
 	return nil
 }
 
+func (pk *PrivateKey) parseECDHPrivateKey(data []byte) (err error) {
```

Comment:
This function looks like it's a duplicate of the ones around it. It looks like the only difference between them is the types they cast into? Is that a "don't fix what ain't broke" thing?